.github/workflows/sycl-linux-precommit.yml

```diff
@@ -102,35 +100,27 @@ jobs:
             extra_lit_opts: -j 1
           - name: Intel Arc A-Series Graphics
             runner: '["Linux", "arc"]'
-            image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
             target_devices: level_zero:gpu;opencl:gpu;level_zero_v2:gpu
-            extra_lit_opts: --param matrix-xmx8=True
           - name: E2E tests with dev igc on Intel Arc A-Series Graphics
```

Comment:
And may be "Dev IGC on Intel Arc A-Series Graphics" as it was done for PVC.

Reply:
done